### PR TITLE
5 temperature sensor connection

### PR DESCRIPTION
**Pull request by DjKrys**
#5

### CODE REVIEW

File: docs/PINOUT.MD [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# PINOUT for digital temperature sensor DS18B20
+                                                NUCLEO PIN                SENSOR PIN
+                                                   PC8  ------------------->  DQ
+                                                   +5V  ------------------->  VDD
+                                                   GND  ------------------->  GND
```
Comment (LigasN) [inline docs/PINOUT.MD]: An arrow should not be in the current direction? In the other case, we should make two tips.

File: docs/PINOUT.MD [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+To correct work temperature sensor should be connected also resistor pull up between pin VDD and DQ.
```
Comment (LigasN) [inline docs/PINOUT.MD]: Pull up resistor should be connected between pin VDD and DQ to assure correct work of temperature sensor.

File: docs/PINOUT.MD [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Schematic to correct connection:
```
Comment (LigasN) [inline docs/PINOUT.MD]: *of

Comment (LigasN) [reply docs/PINOUT.MD]: connections

**Review by LigasN — CHANGES_REQUESTED** [review]
It would be awesome if you will move photos as assets into /docs/assets/

File: docs/PINOUT.MD (hunk 2 above)
Comment (DjKrys) [reply docs/PINOUT.MD]: fixed

File: docs/PINOUT.MD (hunk 3 above)
Comment (DjKrys) [reply docs/PINOUT.MD]: fixed

File: docs/PINOUT.MD (hunk 1 above)
Comment (DjKrys) [reply docs/PINOUT.MD]: fixed

**Review by LigasN — APPROVED** [review]

**Pull request merged**